aztk/utils/ssh.py

```diff
@@ -19,6 +17,7 @@ def connect(hostname,
             username=None,
             password=None,
             pkey=None):
+    import paramiko
```

Comment:
why is this necessary?

Reply:
it was being imported on the cluster and we don;t install it on the cluster. Other options is to install on the cluster

Reply:
when/why was it being imported on the cluster?

Reply:
dependency with module 

Traceback (most recent call last):
  File "/mnt/batch/tasks/startup/wd/aztk/node_scripts/main.py", line 2, in <module>
    import aztk.spark
  File "/mnt/batch/tasks/startup/wd/aztk/spark/__init__.py", line 2, in <module>
    from .client import Client
  File "/mnt/batch/tasks/startup/wd/aztk/spark/client.py", line 5, in <module>
    from aztk.client import Client as BaseClient
  File "/mnt/batch/tasks/startup/wd/aztk/client.py", line 15, in <module>
    import aztk.utils.ssh as ssh_lib
  File "/mnt/batch/tasks/startup/wd/aztk/utils/ssh.py", line 12, in <module>
    import paramiko
ImportError: No module named 'paramiko'